omnia/lib/feed.sh

```diff
@@ -24,20 +24,20 @@ readSourcesAndBroadcastAllPriceMessages()  {
 				continue
 			fi
 			local _assetPair=$(jq -r .asset <<<"$_json")
-			_assetPair="${_assetPair^^}"
```

Comment:
I think we should still uppercase asset pair

Reply:
or maybe make sure that we track unpublished pairs without a slash (we wanted to be backward compatible and so we remove the slash from the symbol everywhere in the omnia "core" code, leaving the externals untouched. 
That was the policy for the slashes in omnia

Reply:
I did a #97 to help with what I said

Reply:
lgtm